envoy/config/filter/http/ext_authz/v2/ext_authz.proto

```diff
@@ -23,4 +23,25 @@ message ExtAuthz {
   // is NOT denied then traffic will be permitted.
   // Defaults to false.
   bool failure_mode_allow = 2;
+
+  // When this value is set to true, the filter must call an HTTP authorization service only.
+  bool use_http_service = 3;
+
+  // The external authorization HTTP service configuration. Only applicable when `use_http_service`
+  // is true.
+  message HttpService {
+
+    // Sets the cluster name which the authorization request must be sent to.
+    string cluster = 1;
```

Comment:
you can mandate this be set. see https://github.com/gsagula/data-plane-api/blob/339db0e1225d31e13ea3d35e7a8459abfbda3970/envoy/api/v2/core/grpc_service.proto#L25

Do you think there would ever be a need for a TLS configuration and also could the http service be made a core service (like grpc_service). (Not asking for you to change anything but thinking out loud).

Reply:
TLS config is configured at the cluster level, so I think this is covered.